DragaliaAPI.Shared/Definitions/Enums/UnitDataEnums.cs

```diff
@@ -2,6 +2,7 @@
 
 public enum UnitElement
 {
+    None = -1,
```

Comment:
is this how the game stores it? not aware of any elementless units

Reply:
the puppy